services/core-web/src/actionCreators/mineAlertActionCreator.ts

```diff
@@ -10,76 +10,100 @@ import * as reducerTypes from "@/constants/reducerTypes";
 import { AxiosResponse } from "axios";
 import { AppThunk } from "@/store/appThunk.type";
 
-export const fetchMineAlertsByMine =
-  (mineGuid: string): AppThunk<Promise<AxiosResponse<IMineAlert>>> =>
-  (dispatch): Promise<AxiosResponse<IMineAlert>> => {
-    dispatch(request(reducerTypes.GET_MINE_ALERTS));
-    dispatch(showLoading("modal"));
-    return CustomAxios()
-      .get(`${ENVIRONMENT.apiUrl}${API.MINE_ALERTS(mineGuid)}`, createRequestHeader())
-      .then((response: AxiosResponse<IMineAlert>) => {
-        dispatch(success(reducerTypes.GET_MINE_ALERTS));
-        dispatch(mineActions.storeMineAlerts(response.data));
-        return response;
-      })
-      .catch((err) => {
-        dispatch(error(reducerTypes.GET_MINE_ALERTS));
-        throw new Error(err);
-      })
-      .finally(() => dispatch(hideLoading()));
-  };
+export const fetchGlobalMineAlerts = (): AppThunk<Promise<AxiosResponse<IMineAlert>>> => (
+  dispatch
+): Promise<AxiosResponse<IMineAlert>> => {
+  dispatch(request(reducerTypes.GET_GLOBAL_MINE_ALERTS));
+  dispatch(showLoading());
```

Comment:
This function (fetchGlobalMineAlerts) was added, the rest in this file is formattiing